src/odemis/gui/model/__init__.py

```diff
@@ -1121,6 +1121,9 @@ def __init__(self, microscope):
         # Overview streams
         self.overview_streams = model.VigilantAttribute({})  # dict: int --> stream or None
 
+        # Alignment status
+        self.is_aligned = model.BooleanVA(False)
```

Comment:
Eventually, this should it automatically be reset whenever the user pumps the chamber. Even turning off/on the e-beam? Other cases? Do you have already plans to add these? Maybe you can already do some of it in this PR.